src/Components/Modal/Modal.jsx

```diff
@@ -0,0 +1,248 @@
+// Modal.jsx
+import React, { useState } from "react";
+import styles from "./Modal.module.css";
+import { useFetch } from "../../hooks/useFetch";
+import { BASE_URL, DEPLOY_URL } from "../../constants/baseURL";
+import close from "../../assets/close.svg";
+import check from "../../assets/check.png";
+import kakaotalk from "../../assets/kakaotalk.png";
+import facebook from "../../assets/facebook.png";
+import link from "../../assets/link.png";
+import { toast, ToastContainer } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+function Modal({
+    title,
+    subtitle,
+    input,
+    placeholder,
+    list,
+    btnText,
+    btnColor,
+    share,
+    folderId,
+    onClose,
+    onSubmit,
+}) {
+    const [inputValue, setInputValue] = useState("");
+    const folderList = useFetch(`${BASE_URL}users/1/folders`);
+    const [isError, setIsError] = useState(false);
+    const url = `${DEPLOY_URL}shared?user=1&folder=${folderId}`;
+
+    const handleChange = (e) => {
+        setInputValue(e.target.value);
+    };
+
+    const handleBlur = () => {
+        if (inputValue.trim() !== "") {
+            setIsError(false);
+        } else setIsError(true);
```

Comment:
inputValue.trim() 값이 빈 문자열인 경우에는 false 아닌 경우에는 true를 리턴하고 싶다면 아래와 같이 작성할수도 있답니다! 

```suggestion
            setIsError(!!inputValue.trim());
```